### PR TITLE
Make bitmapfont objects constexpr

### PR DESCRIPTION
**Pull request by fmatthew5876**
* Move to headers
* Add test

### CODE REVIEW

**Comment by fmatthew5876** [comment]
Unfortunately this may be impossible with MSVC 2017. I haven't found any supporting articles, but if you go to `bitmapfont_wqy.h` and `#if 0 .. #endif` all of the initializers after 24110 it compiles.

It looks like a limitation related to the size of the initializer expression because smaller data sets do compile as constexpr.

**Comment by fmatthew5876** [comment]
This appears to build in visual studio 2019.

**Comment by fmatthew5876** [comment]
jenkins test this please

**Comment by carstene1ns** [comment]
This should involve a generator update: https://github.com/EasyRPG/Player/blob/master/resources/shinonome/generate_cxx_font.rb

**Comment by fmatthew5876** [comment]
Updated the ruby script and regenerated the headers

File: src/bitmapfont_wqy.h [hunk 1]
```diff
@@ -1,6 +1,10 @@
-#include "bitmapfont.h"
+#ifndef EP_BITMAPFONT_WQY_H
```
Comment (carstene1ns) [inline src/bitmapfont_wqy.h]: seems like this one is not updated.
(missing `GENERATED FILE` stuff)

Comment (fmatthew5876) [reply src/bitmapfont_wqy.h]: For some reason, this file is not in the ruby script so it's not a generated file??

Comment (fmatthew5876) [reply src/bitmapfont_wqy.h]: I found the python script, updated  and reran it.

**Comment by Ghabry** [comment]
Jenkins: test this please <3

**Pull request merged**